### ISSUE TITLE
 Every first Custom DPL usage fails, using Material-Skin playlist via browser and then Custom DPL

### ISSUE THREAD

**Issue by the-other-martin** [AF-1/lms-dynamicplaylists#8]
## DESCRIBE THE BUG
I'm using several different custom DPLs successfully, e.g. below one for "Lieblingstitel". Every first attempt to start Custom DPL after playing Material-Skin built in playlist "Stilrichtung" fails. Second attempt with any other Custom DPL works as expected.

<br><br>
## TO REPRODUCE
1. Coming from Material-Skin built in playlist "Genre"/"Stilrichtung" (selected through Material-Skin V3.03: "Meine Musik"/"Stilrichtung") - player (e.g. Squeezebox Touch) is playing as expected.

2.  First usage of Custom DPL : After selecting "Play" for the Custom DPL (see example above) in material skin dialog box showing "Play" but without any other information - Custom DPL doesn't start.

3. Selecting "Play" for any other Custom DPL after first fail works as expected: dialog box is showing up with "Starting dynamic playlist: xxxxxxxxx" and Custom DPL plays as expected.After selecting "Play" for the Custom DPL (see example above) in material skin dialog box showing "Play" but without any other information - Custom DPL doesn't start.

<br><br>
## EXPECTED BEHAVIOR
Even the first usage of Custom DPL after using Material-Skin built in playlist "Stilrichtung" should start the selected Custom DPL and play.

<br><br>
## LOG EXCERPT
n/a

<br><br>
## NAME/CODE OF DYNAMIC PLAYLIST

```
-- PlaylistName:03 Lieblingstitel
-- PlaylistGroups: Custom Dynamic Playlists
select tracks.id, tracks.primary_artist from tracks
	join genre_track on
		tracks.id=genre_track.track
	join genres on
		genre_track.genre=genres.id
	left join dynamicplaylist_history on
		tracks.id=dynamicplaylist_history.id and dynamicplaylist_history.client='PlaylistPlayer'
	where
		audio=1
		and dynamicplaylist_history.id is null
		and	genres.name not like '%(%)%'
		and	genres.name not like 'Alternative%'
		and	genres.name not like 'Blues%'
		and	genres.name not like 'Boogie%'
		and	genres.name not like 'Books & Spoken%'
		and	genres.name not like 'Cabaret%'
		and	genres.name not like 'Children''s Music%'
		and	genres.name not like 'Christmas%'
		and	genres.name not like 'Classical%'
		and	genres.name not like 'Country%'
		and	genres.name not like 'Deutsch%'
		and	genres.name not like 'Dixie%'
		and	genres.name not like 'Dub%'
		and	genres.name not like 'Ethno%'
		and	genres.name not like 'Industrial%'
		and	genres.name not like 'Jazz%'
		and	genres.name not like 'Kabarett%'
		and	genres.name not like 'Metal%'
		and	genres.name not like 'Operette%'
		and	genres.name not like 'Schlager%'
		and	genres.name not like 'Soundtrack%'
		and	genres.name not like 'Tanzmusik%'
		and	genres.name not like 'Techno%'
		and	genres.name not like 'Volksmusik%'
	group by tracks.id
	order by random()
	limit 30;
```

<br><br>
## VERSION INFO
* LMS version: Version: 8.3.0 - 1667251155 @ Fri 04 Nov 2022 09:19:49 AM CET (piCorePlayer)
* OS (version): piCore - DE - utf8
* Plugin version: Dynamic Playlists 4.0.3
* Web skin/controller:  Material
* Browser (version): Firefox 108.0.1 (64-Bit)

<br><br>
## SCREENSHOTS
![grafik](https://user-images.githubusercontent.com/44535691/209992526-31e699f5-3bf6-4b8b-8f97-b4b1ffaf0e2b.png)
First usage:
![grafik](https://user-images.githubusercontent.com/44535691/209992727-87fb3074-1e80-4dba-bf1c-d2d53b3854a8.png)
Second usage (with other Custom DPL)
![grafik](https://user-images.githubusercontent.com/44535691/209992824-315a0a6e-c002-4aed-a1e3-98060ab5375f.png)
Playing as expected:
![grafik](https://user-images.githubusercontent.com/44535691/209992975-261bd8ad-2204-4b0b-a2bb-38b688b0aa9f.png)
Playing same playlist as in first try successfully:
![grafik](https://user-images.githubusercontent.com/44535691/209993092-b0176baa-f1e3-482f-956d-26a584621dad.png)

<br><br>
## ADDITIONAL CONTEXT
n/a

<br><br>
## UPLOADS
n/a
<br>

**Comment by AF-1** [comment]
EDIT: I think I found the bug. On it.

The _Play_ notification in Material is just Material confirming the selection you made from a list of options (Play, Add, ...). That's why there's nothing else in that message. That's unrelated to the bug.

P.S. I use **DPL** for 'Dynamic Playlists' (the **plugin**) and **dpl** for dynamic playlist(s), the **actual playlist(s)**. Confusing, I know :-)

**Comment by AF-1** [comment]
Should be fixed in 4.0.4.
Before I close this, I'd appreciate a brief confirmation that the problem no longer occurs in 4.0.4. Thank you.

**Comment by the-other-martin** [comment]
You are a LMS hero! After latest update to "Dynamic Playlists 4.0.4" the DPL works as expected for my custom dynamic playlists!
Many thanks! Martin
